t/enqueue.t

```diff
@@ -0,0 +1,186 @@
+use strict;
+use warnings;
+
+use FindBin;
+use lib "$FindBin::Bin/lib";
+
+use HTFeed::Config qw(get_config);
+use HTFeed::DBTools qw(get_dbh);
+use HTFeed::Queue;
+use HTFeed::Volume;
+use Test::Exception;
+use Test::Spec;
+
+describe "bin/enqueue.pl" => sub {
+
+    sub testvolume {
+        HTFeed::Volume->new(
+            packagetype => 'simple',
+            namespace   => 'test',
+            objid       => 'test'
+        );
+    }
+
+    sub fake_bibdata {
+        my $volume = shift;
+        my $sql = join(
+            " ",
+            "REPLACE INTO feed_zephir_items",
+            "(namespace, id, collection, digitization_source, returned)",
+            "VALUES",
+            "(?,         ?,  'TEST',     'test',              '0')"
+        );
+        get_dbh()->do($sql, {}, $volume->get_namespace, $volume->get_objid);
+    }
+
+    sub enqueue_testvolume {
+        my %params = (
+            # defaults
+            volume => testvolume,
+            status => "ready",
+            # override defaults
+            @_
+        );
+
+        return HTFeed::Queue->new->enqueue(%params);
+    }
+
+    sub get_vol_from_queue {
+        my $volume = shift || testvolume;
+        my $status = shift || "ready";
+
+        my $sql = join(
+            " ",
+            "SELECT *",
+            "FROM   feed_queue",
+            "WHERE  namespace = ?",
+            "AND    id        = ?",
+            "AND    pkg_type  = ?",
+            "AND    status    = ?"
+        );
+
+        get_dbh()->selectrow_hashref(
+            $sql,
+            {},
+            $volume->get_namespace,
+            $volume->get_objid,
+            $volume->get_packagetype,
+            $status
+        );
+    }
+
+    # Run a commandline invocation and check output against provided rx
+    sub enqueue {
+        my $cmd                = shift;
+        my $match_rx           = shift;
+        my $expect_match_count = shift;
+	my $verbose            = shift || 0;
```

Comment:
`0x09` rears its head again thanks to my BBEdit settings. To wit, whatever you did in previous PRs to get tabs expanded to spaces should be done again. I think the `my $verbose` line is the first one in the file where tab indentation starts showing up.